mantle/cmd/ore/aws/upload.go

```diff
@@ -182,7 +182,11 @@ func runUpload(cmd *cobra.Command, args []string) error {
 	s3ObjectPath := strings.TrimPrefix(s3URL.Path, "/")
 
 	if uploadForce {
-		API.RemoveImage(uploadAMIName, s3BucketName, s3ObjectPath)
+		err := API.RemoveImage(uploadAMIName, s3BucketName, s3ObjectPath)
```

Comment:
One doubt I had here was whether the error was being purposely ignored in case the image didn't exist. But looking at the implementation of `RemoveImage`, it looks like that handles the "`ENOENT`" already for the image and snapshot. So I think this is correct.